java/api/src/main/java/io/ray/api/placementgroup/Bundle.java

```diff
@@ -0,0 +1,13 @@
+package io.ray.api.placementgroup;
+
+import io.ray.api.id.PlacementGroupId;
+
+public class Bundle {
```

Comment:
I think we can only expose an empty interface in the API. Users don't need to know the placement group id and the index.

Reply:
Discussed offline. Better rename this to `PlacementBundle` to make it more aligned with `PlacementGroup`.

Reply:
fixed